app/views/podcast_planner/_form_side.html.erb

```diff
@@ -1,56 +1,51 @@
-<div class="card">
-  <div class="card-header-primary">
-    <h5 class="card-title"><%= t(".header.preview") %></h5>
+<%= form_with(
+      url: podcast_planner_path(@podcast),
+      method: :post
+    ) do |form| %>
+  <div class="mb-4">
+    <h3 class="mb-3"><%= t(".header.publish_time") %></h3>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.select :publish_time, time_options %>
+        <%= form.label :publish_time %>
+      </div>
+    </div>
   </div>
 
-  <%= form_with(
-        url: podcast_planner_path(@podcast),
-        method: :post
-      ) do |form| %>
-      <div class="card-body">
-        <div class="row mb-4">
-          <div class="col-lg-6">
-            <h3 class="fw-bold"><%= t(".header.publish_time") %></h3>
-            <div class="form-floating">
-              <%= form.select :publish_time, time_options %>
-              <%= form.label :publish_time %>
-            </div>
-          </div>
-
-          <div class="col-lg-6">
-            <h3 class="fw-bold"><%= t(".header.audio_segments") %></h3>
-            <div class="form-floating">
-              <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2} %>
-              <%= form.label :segment_count %>
-            </div>
-          </div>
-        </div>
-
-        <div class="col-lg">
-          <div data-controller="pager" data-pager-range-value="6">
-            <div class="mb-2">
-              <%= button_tag type: "button", class: "btn btn-primary btn-sm", data: {action: "click->pager#pageBackward", pager_target: "prev"}, disabled: true do %>
-                <span class="material-icons">arrow_left</span>
-                Month
-              <% end %>
-              <%= button_tag type: "button", class: "btn btn-primary btn-sm", data: {action: "click->pager#pageForward", pager_target: "next"} do %>
-                Month
-                <span class="material-icons">arrow_right</span>
-              <% end %>
-            </div>
+  <div class="mb-4">
+    <h3 class="mb-3"><%= t(".header.audio_segments") %></h3>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2} %>
+        <%= form.label :segment_count %>
+      </div>
+    </div>
+  </div>
 
-            <%= turbo_frame_tag "preview", target: "_top" do %>
-              <div class="row">
-                <% 24.times do |i| %>
-                  <%= render "calendar", form: form, start_date: Date.today.beginning_of_month + i.months, planner: @planner, index: i, range: 6 %>
-                <% end %>
-              </div>
-            <% end %>
-          </div>
+  <div class="card p-4 mt-4mb-4">
+    <div data-controller="pager" data-pager-range-value="6">
+      <div class="d-flex justify-content-between">
+        <h3 class="mb-3"><%= t(".header.number_of_drafts") %></h3>
```

Comment:
this needs a `count` variable to pass through to the yml file in order to display the correct number.

you can probably pass through `@planner.dates.count` but that would only reflect the dates generated through the calculator, not selected manually. i could probably figure out a helper method to add to the stimulus controller that would update every time a manual date has been clicked as well. is this an important feature for parity?